response_operations_ui/templates/reporting-unit.html

```diff
@@ -2,161 +2,178 @@
 {% block page_title %}{{ru.name}} | Reporting units | Survey Data Collection{% endblock %}
 
 {% block main %}
-{% include 'partials/info-box.html' %}
-<h1 id="RU_NAME" class="jupiter">{{ ru.name }}</h1>
 
-  <dl class="item-headlines">
-    <dt class="item-headlines-term">
-      Reference:
-    </dt>
-    <dd id="RU_REF" class="item-headlines-definition">
-      {{ ru.sampleUnitRef }}
-    </dd>
-  </dl>
 
-  <h2 class="saturn">Surveys</h2>
+{% if edit_details %}
```

Comment:
This is fine for now but it would probably be better to trigger this with the `info-box` partial as it's doing the same thing just with a different message